build.sbt

```
@@ -79,3 +80,10 @@ lazy val `portable-scala-reflect` = crossProject(JSPlatform, JVMPlatform)
       else prev.filter(v => !v.startsWith("2.10."))
     }
   )
+  .nativeSettings(
+    crossScalaVersions := Seq("2.11.12"),
```

This can now be widened
```suggestion
    crossScalaVersions := Seq("2.11.12", "2.12.13", "2.13.4"),
```